### ISSUE TITLE
Why not use Expression.NotEqual in TagWith?

### ISSUE THREAD

**Issue by d-a-s** [VariableNotFound/ef6-tagwith#11]
I noticed when using an IQueryable's `.ToString()` that the SQL generated by this method is
```
WHERE ... AND (N'!__tag!' = N'My Comment') ...
```
which defaults to being FALSE. Thus if I want to run that query I get from a ToString() I have to change `=` to `<>` before running the query.

I wonder if you'd consider using `Expression.NotEqual` instead? Seems like it'd be just as easy, and would make the where clause be more transparent for `.ToString()` and any other similar situations that might come up.

**Comment by jmaguilar** [comment]
Hello,

First of all, thanks for your feedback :)

I'm going to study it. Definitely, we can not switch to "not equals" for everybody because it could be a breaking change for people that may be relying on the current generated SQL (for example, for unit testing). However, it could be a configuration setting in TaggingOptions or something like that.

I'm a bit busy now, but I'll take a look at it in the next few weeks.

Thanks!

PS: Of course, you are welcome to send a PR with this feature! 😉

**Comment by jmaguilar** [comment]
Hi @d-a-s ,

I've just published the 1.3.0-preview1 NuGet package. With this version you can change the predicate used by default from the component initialization code:

```cs
TagWith.Initialize<SqlServerTagger>(
    new TaggingOptions() { 
        PredicateExpression = PredicateExpression.NotEquals
    }
);
```

Please, take a look at this new version and check if everything works fine :)

Thanks!
